esp/services/ws_fs/ws_fsService.cpp

```diff
@@ -3380,9 +3425,40 @@ bool CFileSprayEx::onDropZoneFiles(IEspContext &context, IEspDropZoneFilesReques
         bool filesFromALinux = false;
         IArrayOf<IEspDropZone> dropZoneList;
         bool ECLWatchVisibleOnly = req.getECLWatchVisibleOnly();
-
 #ifdef _CONTAINERIZED
-        IERRLOG("CONTAINERIZED(CFileSprayEx::onDropZoneFileSearch)");
+        bool isIPAddressReq = !isEmptyString(netAddress) && isIPAddress(netAddress);
+        ForEachItemIn(i, allTpDropZones)
```

Comment:
Could this code be used in both bare metal and containerized code?

Reply:
Yes, I will change it